src/rhsm/connection.py

```diff
@@ -766,6 +768,26 @@ def supports_resource(self, resource_name):
 
         return resource_name in self.resources
 
+    def _load_manager_capabilities(self):
+        """
+        Loads manager capabilities by doing a GET on the status
+        resource located at '/status/'
+        """
+        self.capabilities = {}
+        self.capabilities = self.conn.request_get("/status/")
+        self.capabilities = self.capabilities['managerCapabilities']
+        log.debug("Server has the following capabilities: %s",
+                  self.capabilities)
+
+    def has_capability(self, capability):
+        """
```

Comment:
How many times does this get called in any given run? It is not a dynamic list. see if you can cache the capability list.